### DISCUSSION TITLE
RSSI value cosmetics (AFHDS2A)

### DISCUSSION THREAD

**Discussion by didinski** [OpenI6X/opentx#388] [General]
Hello,

im getting a little bit confused by the RSSI value. (AFHDS2A) 

current status:

1. RSSI value is 75 in the Telemetry screen (Numbers)  
[flysky_ibus.cpp | line 137: 135 - value]

2. RSSI unit is dB           
[flysky_ibus.cpp | line 161]

3. RSSI bar in Telemetry screen (fixed, bottom) shows a value of 99 
 [view.telemetry.cpp | line 38: rssi = min((uint8_t)99, TELEMETRY_RSSI()); ]

[Pic: RSSI fixed bar. bottom. ](https://2672709489-files.gitbook.io/~/files/v0/b/gitbook-x-prod.appspot.com/o/spaces%2FjAgpAXMB9RpRNV8hOqZD%2Fuploads%2FE414wSUP8nftG7acLJIO%2Fbwtel2.png?alt=media&token=aa2a2698-3bec-4cfe-9dd6-018c232715ae)

#3 is the most confusing to me. What I understand: 
The RSSI value is compared to 99. If its lower than 99 the value will we shown on screen.
The maximum value is 99. So the RSSI values have to be in the range 0-99
I thougt the RSSI coming in and is converted from -60 to 75?
So what RSSI value is then equivalent to 0?
I dont understand that „scaling“. Maybe someone can teach me.

Suggestion to:

1. set the RSSI value back to -60

2. use the unit [dBm]

3. use RQLY for the fixed bar. For RSSI you can use the upper bars where the boundaries can be set (RX depending)

**Comment by ajjjjjjjj** [comment]
Suggestion 3. seems like proper solution and will match both AFHDS2A and CRSF behaviour.

**Comment by ajjjjjjjj** [comment]
Label updated to RQly in https://github.com/OpenI6X/opentx/commit/dc1ae6b3d80ca22822d5ba53fcaf3e5c1960455d
